README.md

```diff
@@ -21,6 +21,13 @@ $ cdklocal --version
 
 (Note: Depending on your local setup, you may or may not have to use the global `npm` installation flag `-g` above.)
 
+### Mac OS specific `MODULE_NOT_FOUND` issue
+On Mac OS, brew could be used to install AWS CDK, which will result in a `MODULE_NOT_FOUND` error from `cdklocal`.  
```

Comment:
```suggestion
On Mac OS, brew can be used to install AWS CDK, which might result in a `MODULE_NOT_FOUND` error from `cdklocal`.  
```

nit: Just a bit of rewording since it will depend on their setup and how/where they install the package as well.